src/components/CornerLogo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import SCLogo from '@/components/icons/SCLogo'
+import SealCredLogoText from '@/components/icons/SealCredLogoText'
+
+export default function CornertLogo() {
```

Comment:
Do we need to give this function a name?

Reply:
no, fixed. ty